docs/arcaflow/workflows/expressions.md

```diff
@@ -28,33 +183,298 @@ $.foo.bar
+
+Here is an example of a map with string keys and integer values. The map is stored in a field called `foo` in the root-level object:
+
+```yml
+foo:
+  a: 1
+  b: 2
 ```
-$.foo.0
+
+Given the map shown above, the following expression would yield a value of `2`:
+```JavaScript
+$.foo["b"]
 ```
 
-## Array accessor
+## Functions
 
-Alternative to the dot notation, you can also use the array accessor:
+The engine provides predefined functions for use in expressions.  These provide transformations beyond what is
+available from operators.
 
+Functions:
+
+| function definition                              | return type  | description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|--------------------------------------------------|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `intToFloat(integer)`                            | float        | Converts an integer value into the equivalent floating point value.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `floatToInt(float)`                              | integer      | Converts a floating point value into an integer value by discarding the fraction, rounding toward zero to the nearest integer.<br>Special cases:<br>&nbsp; +Inf yields the maximum 64-bit integer (9223372036854775807)<br>&nbsp; -Inf and NaN yield the minimum 64-bit integer (-9223372036854775808)\n\n"<br><br>For example, `5.5` yields `5`, and `-1.9` yields `-1`"                                                                                                                                      |
```

Comment:
Is that an extra `"` at the end of the line?

Reply:
Yup, it looks like it.  I've already fixed several of those...they are cut-and-paste-o's from propagating the text from the engine code to here.  I'll fix that one, too.